app/scripts/views/mixins/password-prompt-mixin.js

```diff
@@ -83,6 +91,15 @@ define(function (require, exports, module) {
       this.logViewEvent(eventName);
     },
 
+    /**
+     * Determines if user is using an English locale.
+     * @returns {Boolean}
+     * @private
+     */
+    _isEnglishLocale: function () {
+      return this.window.navigator.language.indexOf('en') === 0;
```

Comment:
Is the intent to [only show this if English is the user's preferred language](https://developer.mozilla.org/en-US/docs/Web/API/NavigatorLanguage/language)? An edge case to consider is the user's preferred language could be something other than English, but we don't have that language so English is served instead. The call to `/config` returns the locale served to the user: 

<img width="306" alt="screen shot 2016-08-04 at 09 40 43" src="https://cloud.githubusercontent.com/assets/848085/17395733/d0113d2c-5a27-11e6-83b3-b50ee6d6625e.png">

Config [is not currently passed into the views](https://github.com/mozilla/fxa-content-server/blob/fcf1ebf85de833c1098eb6685a17bae4bf3d4a08/app/scripts/lib/app-start.js#L511), but no reason it can't be.